### ISSUE TITLE
Check and fix: A11y

### ISSUE THREAD

**Issue by const8ine** [open-genes/open-genes-frontend#14]
We have to improve accessibility on the website. Let's discuss if anything is missing on a checklist.

- [ ] Add `alt` attribute to `img`
- [ ] Add `aria` attributes
- [ ] Check navigation with the `Tab` key — does it work and do users see a frame on the focused element while navigating. Fix it.
